### PR TITLE
Add ZooKeeper operator docs

### PR DESCRIPTION
**Pull request by cbuto**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-82302

## Description of changes being made

Adding docs about the zookeeper operator in the workspace catalog.

There are a few additional changes that are needed but opening this to start the review process.

1. Copy this page to 2.2 (needs https://github.com/mesosphere/dcos-docs-site/pull/3956)
2. Add project level docs (will be done in another PR)

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/zookeeper-operator/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    ```bash
+    kubectl get zookeeperclusters -A
+    ```
+
+1.  Deleting a `ZookeeperCluster`:
+
+    ```bash
+    kubectl delete zookeepercluster <name of zookeepercluster>
+    ```
```
Comment (cwyl02) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/zookeeper-operator/index.md:39]: Should we mention this in the CR docs and then provide link to CR docs here?

Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/zookeeper-operator/index.md:39]: yeah we definitely can, I wasn't sure how much detail about uninstalling should go here. maybe the  `View ZookeeperClusters in all namespaces:` should also go in the CR docs

Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/zookeeper-operator/index.md]: from our slack convo, we decided that the CR docs should go in this PR for completeness so I added those in  0e0547a123ef471771aaddce5476347569f13ed6 (along with this suggestion ^)

Comment (cwyl02) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/zookeeper-operator/index.md]: I think anything related to CR can go to CR docs and we just do a link here, commands about CRDs though should stay.

Comment (cbuto) [reply pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/zookeeper-operator/index.md]: yup agreed, fixed in 0e0547a123ef471771aaddce5476347569f13ed6

**Review by cmastr — APPROVED** [review]

**Comment by cbuto** [comment]
moved to the 2.1.1 release branch https://github.com/mesosphere/dcos-docs-site/pull/3971

**Pull request closed without merging**